custom_components/mikrotik_router/config_flow.py

```diff
@@ -131,6 +137,7 @@ def _show_config_form(
                     vol.Optional(CONF_PORT, default=port): int,
                     vol.Optional(CONF_NAME, default=name): str,
                     vol.Optional(CONF_SSL, default=use_ssl): bool,
+                    vol.Optional(CONF_TRACK_ACCOUNTING, default=track_accounting): bool,
```

Comment:
Adding Enable accounting would be better in integration options like other parameters (in _show_config_form), not in integration setup.